downstreamadapter/dispatcher/dispatcher.go

```diff
@@ -277,13 +277,13 @@ func (d *Dispatcher) HandleEvents(dispatcherEvents []DispatcherEvent, wakeCallba
 
 		event := dispatcherEvent.Event
 		// Pre-check, make sure the event is not stale
-		if event.GetCommitTs() < atomic.LoadUint64(&d.resolvedTs) {
+		if event.GetCommitTs() < d.GetResolvedTs() {
 			log.Info("Received a stale event, should ignore it",
```

Comment:
can this be a verbose log ?

Reply:
Under normal circumstances, this situation should not occur; if it does, it indicates some anomaly, though it is not a correctness issue. Therefore, I am logging it here for further investigation later.

Reply:
how about set to `warn` level ?